src/lib/utils/urls.ts

```diff
@@ -34,3 +35,16 @@ export function buildConnectorIconURL(wallet: Wallet): string {
     import.meta.url
   ).href;
 }
+
+export function buildNetworkIconUrlV2(network: Network | string): string {
+  const networkName =
+    typeof network === 'string' ? network : getNetworkIconName(network);
+
+  const result = new URL(
+    // https://vitejs.dev/guide/assets.html#new-url-url-import-meta-url
+    // Warning: Don't extract this template into a variable or it will stop working in production builds
+    `/src/assets/images/icons/networks-2/${networkName}.svg`,
+    import.meta.url
+  ).href;
+  return result;
+}
```

Comment:
Why do we need a v2 instead of refactoring v1?

Reply:
I created this, bc there were different icons in design. will remvoe